capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -224,6 +224,14 @@ For example if you need to show animated illustrations you can use a `Lottie
 
 You can find more details `here <customization-guide.html>`_.
 
+QR code scanning
+~~~~~
+
+Breaking Changes
+++++++++++++++++
+
+QR code scanning UI and functionality are changed. Scanning and processing happens automatically. You can find more details `here <features.html#qr-code-scanning>`_.
```

Comment:
[Request] The section you link to in `features.rst` was not updated and still says that the user has to tap a popup. Can you please also update that?